libs/membership/website/src/lib/has-active-subscription.tsx

```diff
@@ -0,0 +1,22 @@
+import {useUser} from '@wepublish/authentication/website'
+import {useSubscriptionsQuery} from '@wepublish/website/api'
+import {useMemo} from 'react'
+
+export const useHasActiveSubscription = () => {
+  const {hasUser} = useUser()
+  const {data} = useSubscriptionsQuery({
+    fetchPolicy: 'cache-first',
+    skip: !hasUser
+  })
+
+  return useMemo(
+    () =>
+      !!data?.subscriptions.find(
+        subscription =>
```

Comment:
This logic is implemented manually all over the code. We could re-use it in a dedicated function.

Reply:
This is the dedicated function.